src/views/preview/preview.jsx

```diff
@@ -525,8 +531,11 @@ const mapStateToProps = state => {
 };
 
 const mapDispatchToProps = dispatch => ({
-    handleDeleteComment: (projectId, commentId, token) => {
-        dispatch(previewActions.deleteComment(projectId, commentId, token));
+    handleAddComment: (comment, topLevelCommentId) => {
+        dispatch(previewActions.addNewComment(comment, topLevelCommentId));
```

Comment:
It's a little weird right now with adding comments done partly in the component and partly in redux, while both the request and the state manipulation is done in redux for deleting comments. Reading the file right here, it's confusing that one needs a token but the other doesn't.  Are you planning to move the comment add request into Redux?